plugins/xivo-cisco-sccp/common/common.py

```diff
@@ -209,10 +237,17 @@ def __init__(self, app, plugin_dir, gen_cfg, spec_cfg):
         fetchfw_helper = FetchfwPluginHelper(plugin_dir, downloaders)
 
         self.services = fetchfw_helper.services()
+
+        # Maybe find a way to bind to a specific port without changing the general http_port setting of xivo-provd ?
```

Comment:
Another solution would be to add some reverse-proxy configuration in Nginx to redirect HTTP port 6970 to port 8667.

Reply:
There is no solution to send via DHCP another port for the provisioning?
Thank you for the PR.

Reply:
Well i'm not pretty sure, but it seems not :(
I found nothing like that in Cisco docs whether firmwares or call manager updates releases notes.